src/components/CryptoList.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {useEffect, useState} from 'react';
+import {Text} from 'react-native';
+import AntDesign from 'react-native-vector-icons/AntDesign';
+import {Props} from '../interfaces/Interfaces';
+import NoCryptos from './NoCryptos';
+import {
+  HorizontalView,
+  ImageCrypto,
+  ListCryptos,
+  TextChange,
+  TextPrice,
+  ViewEachCryptos,
+  ViewHorizontal,
+  ViewImageandText,
+  ViewPrices,
+  FlatListCustom,
+} from '../styles/CryptoListStyles';
+
+function CryptoList({cryptos}: Props): JSX.Element {
+  const [isSometing, setIsSometing] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (cryptos.length > 0) {
+      setIsSometing(true);
+    } else {
+      setIsSometing(false);
+    }
+  }, [cryptos]);
+
+  return (
+    <FlatListCustom
+      data={cryptos}
+      keyExtractor={(item: {id: number}) => item.id}
+      ListEmptyComponent={isSometing ? null : <NoCryptos />}
```

Comment:
It may not be necessary to create a state for this, try setting cryptos.length > 0 ? directly, it probably works and saves you a few lines of code